### PR TITLE
Making post

### PR DESCRIPTION
**Pull request by tanguyguillo**
Creating utilisation twig and show posts

@castroCrea

### CODE REVIEW

File: app/controllers/HomepageController.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+namespace Application\Controllers\HomepageController;
+
+use Application\Controllers\Controller;
+
+require_once(ROOT . '/app/controllers/controller.php'); //
```
Comment (castroCrea) [inline app/controllers/HomepageController.php:6]: Normally you don't need that. check how to remove the `require` and use the `use` instead

File: app/lib/database.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?php
+namespace
+Application\Lib\Database;
```
Comment (castroCrea) [inline app/lib/database.php:3]:
```suggestion
namespace Application\Lib\Database;
```

**Review by castroCrea — CHANGES_REQUESTED** [review]
good job

Just change the require

**Comment by tanguyguillo** [comment]
#6

**Comment by tanguyguillo** [comment]
I did a lot of change on the organisation of files with one for model, one for controllers, one for view (twig) on for config, ne for core (database)... posts listing working and detail page working

**Comment by tanguyguillo** [comment]
Just missing author on detail page... and  button comment and comment.... etc...

File: app/controllers/HomepageController.php (hunk 1 above)
Comment (tanguyguillo) [reply app/controllers/HomepageController.php:6]: Yes a know but here the use is recognized by VS code but it's not working, for now i don't know why????


Fatal error: Uncaught Error: Class "Application\Controllers\Controller" not found in /Users/Tanguy/Documents/sites/blog-omega/app/controllers/HomepageController.php:11 Stack trace: #0 /Users/Tanguy/Documents/sites/blog-omega/public/index.php(13): require_once() #1 {main} thrown in /Users/Tanguy/Documents/sites/blog-omega/app/controllers/HomepageController.php on line 1
Capture d’écran 2022-07-05 à 08 30 36
Capture d’écran 2022-07-05 à 08 30 36
1

**Review by castroCrea — APPROVED** [review]

**Comment by castroCrea** [comment]
Tu peux merge ça en cliquan ici 
image

**Comment by tanguyguillo** [comment]
i did it

[image: Capture d’écran 2022-07-11 à 11.08.33.png]


Le dim. 10 juil. 2022 à 22:39, castroCrea ***@***.***> a
écrit :

File: app/controllers/HomepageController.php (hunk 1 above)
Comment (tanguyguillo) [reply app/controllers/HomepageController.php:6]: It's ok know by coding a special autoloading which fix this isuue

**Pull request merged**